test/test_other.py

```diff
@@ -9730,6 +9730,13 @@ def test_noderawfs_access_abspath(self):
     self.run_process([EMCC, 'access.c', '-sNODERAWFS'])
     self.run_js('a.out.js', args=[os.path.abspath('foo')])
 
+  def test_noderawfs_readfile_prerun(self):
+    create_file('foo', 'bar')
+    create_file('pre.js', r'''
+      Module.preRun = () => console.log(FS.readFile('foo', { encoding: 'utf8' }));
+    ''')
```

Comment:
Oh, we also have utility for this: `self.add_pre_run("console.log(FS.readFile('foo', { encoding: 'utf8' })")`

That will also add `'--pre-js=pre.js'` for you.